app/_src/kubernetes-ingress-controller/deployment/minikube.md

```diff
@@ -47,6 +47,26 @@ Kubernetes cluster.
    export PROXY_IP=<ip-address>
    echo $PROXY_IP
    ```
+
+1. Verify that you can access {{site.base_gateway}}:
+
+   ```bash
+   curl -i $PROXY_IP
+   ```
```

Comment:
We should show expected output here too. Seeing "no routes configured" may be confusing